inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -230,6 +233,16 @@ def is_submission(obj, eng):
     return False
 
 
+@with_debug_logging
+def is_valid(obj, *args, **kwargs):
+    """Check if the record is schema compliant."""
+    try:
+        if validate(obj.data, 'hep') is None:
+            return True
+    except ValidationError:
+        return False
```

Comment:
You are losing all information about the validation error by doing it like this. Instead, you should somehow log the info about the actual validation error.